api/server/models/user.js

```diff
@@ -281,4 +281,36 @@ module.exports = function (User) {
     );
 //********************************* END FORGET PASSWORD **********************************
 
+//********************************* START RESET PASSWORD **********************************
+    User.reset_password = function ( req, password, callback) {
+        var access_token_userid = req.accessToken.userId;
+        User.findById( access_token_userid, function(err, useruser) {
+            if( err ){
+                callback(null, 0, 'Error', {});
+            }else{
+                useruser.updateAttribute('password', password, function(err, user) {
```

Comment:
do we need to check user object here?
what if invalid access token is there. what happens to user object?